code/modules/arcade/arcade_prize.dm

```diff
@@ -2,19 +2,19 @@
 * Prize balls
 * Prize tickets
 */
-/obj/item/toy/prizeball
+/obj/item/toy/figure/mechball
```

Comment:
Why did you make this type `mechball` when it has other types that arent mechs

![image](https://user-images.githubusercontent.com/25063394/125786385-59b1a234-3d66-4f9d-880a-250cbdb3e221.png)

Reply:
Good point. Seems I overlooked that one. Will change it to something more sensible